src/Raven.Client/Documents/Linq/LinqPathProvider.cs

```diff
@@ -491,6 +492,12 @@ public static bool IsCounterCall(MethodCallExpression mce)
                    || mce.Object?.Type == typeof(ISessionDocumentCounters) && mce.Method.Name == "Get";
         }
 
+        public static bool IsTimeSeriesCall(MethodCallExpression mce)
+        {
+            return mce.Method.DeclaringType == typeof(ITimeSeriesQueryable) ||
```

Comment:
Do we need `ITimeSeriesLoadQueryable` also here?

Reply:
this can happen only if the expression ends with:
```.Where((ts, tag) => ...)```  which is not valid (missing ToList)
added a check to catch this case and other cases with wrong return type + tests